plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -2169,220 +2169,21 @@ insertNewDefinitionTests = testGroup "insert new definition actions"
 addFunctionArgumentTests :: TestTree
 addFunctionArgumentTests =
   testGroup
-    "add function argument"
-    [ testSession "simple" $ do
```

Comment:
Some of these tests seem to have vanished? Is that deliberate/are the new tests equally good?